pkg/operator/k8sutil/endpoint.go

```diff
@@ -20,22 +20,22 @@ import (
 	"context"
 	"fmt"
 
-	v1 "k8s.io/api/core/v1"
+	discoveryv1 "k8s.io/api/discovery/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 )
 
 // CreateOrUpdateEndpoint creates a service or updates the service declaratively if it already exists.
```

Comment:
```suggestion
// CreateOrUpdateEndpoint creates a EndpointSlice or updates the EndpointSlice declaratively if it already exists.
```
Its not directly related of your changes, but I think this might be a good time to correct the method description

Reply:
Updated.